k8s/alpha/slactwin-db-deployment.yml

```diff
@@ -34,11 +34,11 @@ spec:
                   key: slactwin-db-uri
           resources:
             requests:
-              memory: "4Gi"
-              cpu: "1"
+              memory: "16Gi"
+              cpu: "8"
```

Comment:
probably not necessary as there is only open tornado server.

Reply:
1f4dd11

Reply:
https://github.com/slaclab/slactwin/pull/32/commits/1f4dd11c8d5b90d6769ca226b2f1b3db3accf43e